syntax = "proto3";

service ClearAndSmoothService {
    rpc ClearAndSmooth (ClearAndSmoothRequest) returns (ClearAndSmoothResponse);
}

message ClearAndSmoothRequest {
    repeated double data = 1;
}

message ClearAndSmoothResponse {
    repeated double smoothed_data = 1;
}